documentation/SRS.md

```diff
@@ -0,0 +1,83 @@
+# Human resource management web application
+Application should provide:
+- Storing departments and employees in a database;
+- Display a list of departments and the average salary for each department;
+- Display a list of employees with an indication of the employee's salary;
+- Searching employees born on a certain date or in the period between dates;
+- Updating a list of employees (adding, editing, removing);
+- Updating a list of departments (adding, editing, removing).
+## Departments
+### Display a list of departments
+The mode is designed to view a list of departments, average salary of departments, and also edit the department's
+info.
+![Main page](mockups/main_page.png)
+#### Add department
+**Main scenario:**  
+- User clicks the “ADD NEW DEPT” button in the list of departments view mode;
+- Application displays form to enter department's title;
+- User enters department's title and presses “Save” button;
+- New department record is successfully added, then the list of departments with added record is displaying.
+![Add_new_dept](mockups/add_new_dept.png)
```

Comment:
There is still bad formatting. Please fix the indent by adding empty line before each new block.